### PR TITLE
Migrate some CELO Exchange screens to be functional components

### LINKED ISSUE #1862: Migrate ExchangeTradeScreen and ExchangeReviewScreen to be function components

### PR DESCRIPTION
**Pull request by gnardini**
### Description

I will be working with these screens in the short term to allow exchanging any stablecoin. This will help delete old code and reduce some tech debt. However, I couldn't use any hooks because they were still class components.

This PR just moves the two main screens missing in the Exchange flow to function components. No logic changes were made.

### Other changes

N/A

### Tested

Manually and with unit tests.

### How others should test

Test exchanging normally with cUSD/cEUR and make sure everything works.

### Related issues

- Fixes #1862

### Backwards compatibility

N/A

### CODE REVIEW

**Review by kathaypacific — APPROVED** [review]
🚀 awesome!!

File: packages/mobile/src/exchange/__snapshots__/ExchangeTradeScreen.test.tsx.snap [hunk 1]
```diff
@@ -329,7 +348,7 @@ exports[`ExchangeTradeScreen renders correctly 1`] = `
             >
               
               ₱
-              12.09
+              181.81
```
Comment (jeanregisser) [inline packages/mobile/src/exchange/__snapshots__/ExchangeTradeScreen.test.tsx.snap:351]: just checking, is this new value expected?

**Review by jeanregisser — APPROVED** [review]
Thank you so much, sir! 🥇

File: packages/mobile/src/exchange/__snapshots__/ExchangeTradeScreen.test.tsx.snap (hunk 1 above)
Comment (gnardini) [reply packages/mobile/src/exchange/__snapshots__/ExchangeTradeScreen.test.tsx.snap:351]: Yes, it surprised me too, along with the loading above being removed. I think it was due to how the test was loading stuff before but doing the math this looks correct.

This is the exchange rate between cUSD and CELO.

To get this, first we need to convert CELO to cUSD and then cUSD to PHP.

CELO to cUSD is `9.09090909` and cUSD to PHP is '20'
So 1 CELO = `₱181.8181818`

No clue where the 12.09 came from, but 181 is the value I would expect here

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above task on Latest Android internal Build and Latest iOS testflight build V:1.26.0 and updated status in Testrails

**Test rail link:**
**Android 11:**
https://valoraapp.testrail.io/index.php?/tests/view/9908&group_by=cases:section_id&group_order=asc&group_id=35

**iOS 14:**
https://valoraapp.testrail.io/index.php?/tests/view/9782&group_by=cases:section_id&group_order=asc&group_id=35

**Observation :** We are able to buy and sell CELO seamlessly

**Devices :** iPhone 12(14.7.1), Google Pixel 2XL(11.0)

**CC :** @gnardini 

Thanks..!

**Comment by ValoraQA** [comment]
Hey @MuckT we have verified the above task on Latest Android internal Build V:1.26.0 and updated status in Test rails

**Test rail link:**
**Android 8:**
https://valoraapp.testrail.io/index.php?/tests/view/9845&group_by=cases:section_id&group_order=asc&group_id=35

**Android 10:**
https://valoraapp.testrail.io/index.php?/tests/view/9971&group_by=cases:section_id&group_order=asc&group_id=35

**Devices:** Samsung Galaxy A8+(8.0), Samsung Galaxy M40(10.0)

**CC:** @gnardini 

Thanks..!

**Comment by ValoraQA** [comment]
Hey @MuckT. we have verified the above task on Latest Test flight Build V1.26.0 and updated status in Test rails. User is able to buy and sell CELO.

**Test rail link:**
**iOS 13:**
https://valoraapp.testrail.io/index.php?/tests/view/10037&group_by=cases:section_id&group_order=asc&group_id=35

**Devices:** iPhone 7(13.3)

**CC:** @gnardini 

Thanks..!!

**Pull request merged**